docs/courses/design/lesson-flow.md

```diff
@@ -0,0 +1,79 @@
+  Descriptions, examples. R exercise for testing interactions.
+  - Lesson 2.3: EQUIVALENCE, NON-INFERIORITY  
+  Descriptions, examples. MCQ on interpretation.
+  - Lesson 2.4: BIOEQUIVALENCE TRIALS 
+  Descriptions, examples. PK, concentration profiles, AUC. R exercises to plot concentration profiles and generate PK summary stats.  
+ - Chapter 3: SAMPLE SIZE AND POWER
+   - Lesson 3.1: FUNDAMENTALS   
+  Definitions, relationships among effect size, alpha, beta, etc. Mean difference, proportions. R exercises for mean difference and proportions.
+   - Lesson 3.2: ADJUSTMENTS    
+  Unequal group sizes, unequal variances, loss to FU. R exercises.
+   - Lesson 3.3: INTERIM ANALYSES AND GROUP SEQUENTIAL DESIGNS  
+  Descriptions, examples. R exercise.
+   - Lesson 3.4: ALTERNATIVE TRIAL DESIGNS  
+  Descriptions, examples for cluster randomized trials, time-to-event, equivalence. R exercise.  
+  - Chapter 4: STATISTICAL ANALYSES  
+    - Lesson 4.1: ANALYSIS SETS AND SUBGROUPS   
+    ITT and PP. Subgroups. R exercise for subgroups.  
+    - Lesson 4.2: COVARIATES  
+  Confounding. Adjustments for baseline, covariates. Regression analysis. R exercise for regression.  
+    - Lesson 4.3: INTERACTIONS 
+    Description, examples.  R exercise. MCQ for interpretation.  
+    - Lesson 4.4: MULTIPLICITY  
+  Multiple endpoints, combined endpoints, alpha adjustments.  R exercises.
+    - Lesson 4.5: FURTHER TOPICS  
+   Time-to-event data, longitudinal data, missing data. Last handout slide for links to ICH E9, R packages, R guidance for regulated CTs.
+
+## FAQS
+
+### How much detail to I need for each lesson?
+
+You need enough detail that you and your Curriculum Lead will be convinced that there is a sensible amount of interesting content in the lesson, but not so much detail that the flow of the course is obscured. In practice, you can either write a sentence to describe the lesson contents, or you can provide sub-bullets for each exercise that the lesson will contain.
+
+## Good ideas
+
+### Rework Step 3 as you go
+
+By the time you get to this step, you should have a reasonable sense of what topics you want to teach in what order. You may not have felt like that when you started step 3. As you describe the flow of the course, don't feel bound to make it match the example exercises that you created. If you can think of a better flow, wrtie that here and reorder cor change the exercises in Step 3.
+
+## Common problems and their solutions
+
+### Too much content
+
+In order to standardize on course size there are limits on the number of exercises that you can include in each chapter. Each lesson contains a video, and videos take a lot more effort to create than coding exercises. Err on the side of having fewer lessons. 
+
+
+## How will this be reviewed?
+
+This step will be reviewed by at least two people other than your Curriculum Lead, possibly including an external reviewer. They will look for the following points.
+
```

Comment:
Agreed, but I want to keep it fuzzy just for now, since the reviewing policy is in flux. I've creating an issue to remind me to update it later. https://github.com/datacamp/authoring/issues/82